src/Lecture2.hs

```diff
@@ -74,7 +84,10 @@ return the removed element.
 >>> removeAt 10 [1 .. 5]
 (Nothing,[1,2,3,4,5])
 -}
-removeAt = error "TODO"
+removeAt :: (Ord a) => Int -> [a] -> (Maybe a, [a])
```

Comment:
I see that the `Ord a` constraint is redundant here. You don't compare values with each other so you can remove this constraint and make this function work for more cases 🙂